source/Preferences.cpp

```diff
@@ -49,6 +49,21 @@ namespace {
 	const vector<string> VSYNC_SETTINGS = {"off", "on", "adaptive"};
 	int vsyncIndex = 1;
 
+	const vector<string> STATUS_OVERLAYS_ALL = {"off", "always on", "damaged", "--"};
```

Comment:
Since the options will always be the same for all preferences, one vector would seem to be enough.